pages/admin/permit-holder/[permitHolderId].tsx

```diff
@@ -2,94 +2,162 @@ import { GetServerSideProps } from 'next'; // Get server side props
-    province: Province.Bc,
-    relationship: 'Parent',
-  },
-  createdAt: new Date().toDateString(),
-  expirationDate: new Date().toDateString(),
-  isRenewal: true,
-  applicationStatus: ApplicationStatus.Pending, // Will be updated in the future as we'll be able to reference ApplicationStatus enum in types.ts
-  permitFee: 5,
-  donation: 10,
-  paymentType: PaymentType.Visa,
+export type ApplicantData = {
+  id: number;
+  rcdUserId?: number;
+  firstName: string;
+  lastName: string;
+  gender: Gender;
+  dateOfBirth: Date;
+  email?: string;
+  phone: string;
+  province: Province;
+  city: string;
+  addressLine1: string;
+  addressLine2?: string;
+  postalCode: string;
+  status?: ApplicantStatus;
+};
+
+export type PermitData = {
+  rcdPermitId: number;
+  expiryDate: Date;
+  applicationId: number;
+  isRenewal: boolean;
+  status: ApplicationStatus;
+};
+
+export type PermitHolderAttachedFile = {
+  associatedApp?: number;
+  dateUploaded: Date;
+  fileUrl: string;
+};
+
+export type MedicalHistoryEntry = {
+  disablingCondition: string;
+  dateUploaded: Date;
+  associatedApplication: {
+    disability: string;
+    affectsMobility: boolean;
+    mobilityAidRequired: boolean;
+    cannotWalk100m: boolean;
+    aid: Aid[];
+    createdAt: Date;
+  };
+};
+
+export type PreviousPhysicianData = {
+  name: string;
+  mspNumber: number;
+  phone: string;
+};
+
+type Props = {
+  readonly permitHolderId: number;
 };
 
 // Individual permit holder page
-export default function PermitHolder() {
-  // TODO: Destructure physician, guardian from application
-  const { applicant, applicationStatus } = mockApplication;
+export default function PermitHolder({ permitHolderId }: Props) {
+  const [applicantData, setApplicantData] = useState<ApplicantData>();
+  const [permits, setPermits] = useState<PermitData[]>();
+  const [medicalHistoryData, setMedicalHistoryData] = useState<MedicalHistoryEntry[]>();
+  const [attachedFiles, setAttachedFiles] = useState<PermitHolderAttachedFile[]>();
+  const [previousPhysicianData, setPreviousPhysicianData] = useState<PreviousPhysicianData[]>();
+
+  const { data } = useQuery<GetPermitHolderResponse, GetPermitHolderRequest>(GET_PERMIT_HOLDER, {
+    variables: {
+      id: permitHolderId,
+    },
+    onCompleted: data => {
+      setApplicantData({
+        id: data.applicant.id,
+        rcdUserId: data.applicant.rcdUserId || undefined,
+        firstName: data.applicant.firstName,
+        lastName: data.applicant.lastName,
+        gender: data.applicant.gender,
+        dateOfBirth: data.applicant.dateOfBirth,
+        email: data.applicant.email || undefined,
+        phone: data.applicant.phone,
+        province: data.applicant.province,
+        city: data.applicant.city,
+        addressLine1: data.applicant.addressLine1,
+        addressLine2: data.applicant.addressLine2 || undefined,
+        postalCode: data.applicant.postalCode,
+        status: data.applicant.status || undefined,
+      });
```

Comment:
Could we possibly just do this?
```suggestion
      setApplicantData(data.applicant);
```